src/modules/admin/dictionaries/components/dictionaries-table/DictionariesTable.scss

```diff
@@ -0,0 +1,11 @@
+@import 'styles/mixins';
+
+.dictionariesTable {
+  width: 1200px;
+
+  & > table {
+    tr > td:nth-child(3) > div {
```

Comment:
wouldn't it be easier to simply add a new className? This selector is super confusing and nth-child selector isn't necessarily the best idea, ever

Reply:
Adding dedicated classNames which will be available globally or adding suppa not needed - variables to config only for change one cell will me more confusing. Its normal selector.